code/game/machinery/cryopod.dm

```diff
@@ -472,6 +474,28 @@ var/global/list/frozen_items = list("Alpha"=list(),"Bravo"=list(),"Charlie"=list
 			//Despawning occurs when process() is called with an occupant without a client.
 			add_fingerprint(M)
 
+/obj/machinery/cryopod/verb/return_to_lobby()
+	set name = "Return to Lobby"
+	set category = "Object"
+	set src in oview(1)
+	if(occupant != usr)
```

Comment:
Typecast `usr` here, please. Do we want to send back anything besides humans?
You are running some checks there like the mind one that is not necessarily defined for every possible `usr`.